CONTRIBUTING.md

```diff
@@ -12,3 +12,16 @@ for general information about the project.
 
 After making any changes to .proto files make sure to generate all
 implementation by running `make gen-all`.
+
```

Comment:
Maybe this section should also align with [`Compatibility With Original pprof`](https://github.com/open-telemetry/oteps/blob/main/text/profiles/0239-profiles-data-model.md#compatibility-with-original-pprof) in the data model.